### DISCUSSION TITLE
Undefined type Time.exit status 1

### DISCUSSION THREAD

**Discussion by drewwells** [99designs/gqlgen#1808] [Q&A]
### What happened?
Took the starting example an added a new type

### What did you expect?
exit 0 and generated code
### Minimal graphql.schema and models to reproduce
```
type Oracle {
  timestamp: Time!
  price: Int!
  block: Int!
}%
```
```
go run github.com/99designs/gqlgen generate                                          
failed to load schema: graph/oracle.graphqls:7: Undefined type Time.exit status 1
```
### versions
 - `gqlgen version`?          github.com/99designs/gqlgen v0.13.0
 - `go version`? 1.15.6
 - dep or go modules? go

**Comment by Petercopter** [comment]
You want to add

```
scalar Time
```
to your schema, and then it will generate. Took me awhile to figure it out.

**Reply by benjamindally** [reply]
Would love to know more about this. When I put that in, all it does is say that there is `graph/schema.graphqls:38: Undefined type Position.` Are there more steps or did you just put that at the bottom, run generate and it was good to go?
